openapi/index.yaml

```diff
@@ -35,10 +35,102 @@ paths:
           description: Success.
         '500':
           description: Internal Server error
+  /notify:
+    post:
+      operationId: notify
+      summary: Notify a user
+      description: Post the notification to the user using a push notification.
+      parameters:
+        - in: body
+          name: body
+          required: true
+          schema:
+            $ref: '#/definitions/NotificationInfo'
+      responses:
+        '201':
+          description: Success.
+        '400':
+          description: Bad request
+          schema:
+            $ref: '#/definitions/ProblemJson'
+        '401':
+          description: Token null or invalid.
+        '500':
+          description: There was an error in forwarding the notification.
+          schema:
+            $ref: '#/definitions/ProblemJson'
 consumes:
   - application/json
 produces:
   - application/json
+definitions:
+  NotificationType:
+    type: string
+    x-extensible-enum:
+      - MESSAGE
+      - REMINDER_READ
+      - REMINDER_PAYMENT
```

Comment:
we miss the "Reminder pagamento a ridosso scadenza".
We could add something like: "REMINDER_PAYMENT_TOMORROW"

Reply:
Right, added `REMINDER_PAYMENT_LAST` thanks